app/models/noise.rb

```diff
@@ -149,13 +139,17 @@ def media_urls
   end
 
   def local_media_urls
-    @local_media_urls ||= media_entities.collect { |m| m.try(:[], 'media_url_https') ||  m.try(:[], 'media_url_http') }
+    @local_media_urls ||= media_entities.collect do |m|
+      m.try(:[], 'media_url_https') ||  m.try(:[], 'media_url_http')
+    end
   end
 
   def external_media_urls
     @external_media_urls ||= begin
       urls = url_entities.collect { |m| m.try(:[], 'expanded_url') }
-      media_urls ||= urls.collect { |url| url += 'media/?size=l' if url.start_with?('http://instagram.com/', 'https://instagram.com/') }
+      media_urls = urls.collect do |url|
+        url += 'media/?size=l' if url.start_with?('http://instagram.com/', 'https://instagram.com/')
```

Comment:
Useless assignment to variable - `url`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.